packages/components/docs/introduction.story.mdx

```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Introduction" />
+
+# Introduction
+
+Hello World
```

Comment:
Are we concerned about the duplication in terms of docs, this will introduce. Components have READMEs that are currectly synced into the WordPress handbook and they'll have special docs that will live in the storybook?

Reply:
These are the design system docs.

A lot of that content will probably be moved here though.

Reply:
> These are the design system docs.

This is what the README are for. so the components part for the design system will get duplicated?

Reply:
Yes, but MDX is much better than a simple README.md and will be the new framework for them.

I guess we can delete the READMEs once the new docs are fleshed out.

Reply:
We can't delete the READMEs until we figure out how to show Storybook in the wp.org devdocs. I'm concerned that it will take a lot of time and that we'll have to maintain duplicated content for a long time.

Reply:
Storybook can export static files, that won't be hard to do. And it won't be duplicated, the new docs will be better and there won't necessarily be a 1 to 1 mapping, because the goal is to overhaul this package into a proper design system component kit.